src/diffusers/models/attention_processor.py

```diff
@@ -4283,6 +4283,79 @@ def __init__(self):
         super().__init__()
```

Comment:
This picture above needs to be hosted somewhere else (you could consider using the Hugging Face Hub). Also, on our documentation, we prefer not to add human subjects.

Reply:
Sorry, I didn't get your point. What do you mean by "this picture"?

Reply:
The picture added to this PR.

Reply:
Oh. It is for testing the correctness of my implementation. I forgot to remove it. Sorry.